tests/test_interactive.config

```diff
@@ -1 +1 @@
-CONFIG_MODULE_TEST_UTILS_INTERACTIVE_SYNC=y
+CONFIG_MODULE_TEST_UTILS_INTERACTIVE_SYNC=N
```

Comment:
Not sure if it got in my mistake or you wanted to do it only for the test application for this driver?

Reply:
I wanted to test it on iot-lab.info, but i got no success 
`
at30tse75x init 0
Error initializing AT30TSE75x sensor on I2C #0 @ 0x48
` 
knoten: samr21-10.saclay...

Reply:
of cause this line is wrong